src/lib/stitching/exchange/schema.ts

```diff
@@ -17,12 +17,35 @@ export const executableExchangeSchema = () => {
   })
 
   // Return the new modified schema
-  return transformSchema(schema, transformsForExchange)
+  return transformSchema(schema, transforms)
   // Note that changes in this will need to be
 }
 
+const eigenWhitelist = [
+  "OrderWithMutationSuccess",
+  "OrderWithMutationFailure",
+  "CreateOrderWithArtworkInput",
+  "OrderWithMutationSuccess",
+]
+
 export const transformsForExchange = [
   // Apply a prefix to all the typenames
+  new RenameTypes(name => {
+    if (eigenWhitelist.includes(name)) {
+      return name
+    }
+    return `Commerce${name}`
+  }),
+  // Rename all the root fields to be camelCased
+  new RenameRootFields(
+    (_operation, name) =>
+      `commerce${name.charAt(0).toUpperCase() + name.slice(1)}`
+  ),
+]
+
+export const legacyTransformsForExchange = [
+  // Apply a prefix to all the typenames
+  // for Legacy merged schema approach
```

Comment:
Shouldn't this skip the `eigenWhitelist` ones when stitching exchange is disabled?

Reply:
Nope, I'm reasonably sure - because they are already working right now